sourced/ml/models/model_converters/base.py

```diff
@@ -0,0 +1,128 @@
+import logging
+import multiprocessing
+import os
+from pathlib import Path
+from typing import Union
+
+from modelforge import Model
+from modelforge.progress_bar import progress_bar
+
+from sourced.ml.utils.pickleable_logger import PickleableLogger
+
+
+class Model2Base(PickleableLogger):
```

Comment:
Was it copied from the old codebase? What has been changed?

Reply:
Yes, I mentioned it in the PR description. 
Maybe I change few lines to make it work, but nothing more.